utils/utils.go

```diff
@@ -0,0 +1,149 @@
+/*
+ * knoxite
+ *     Copyright (c) 2020, Christian Muehlhaeuser <muesli@gmail.com>
+ *     Copyright (c) 2020, Nicolas Martin <penguwin@penguwin.eu>
+ *
+ *   For license see LICENSE
+ */
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/knoxite/knoxite"
+	"github.com/muesli/crunchy"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var (
```

Comment:
Errors should remain in the default package.

Reply:
Opened a ticket for this #134